src/main/java/org/cascadebot/cascadebot/permissions/objects/PermissionMode.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.permissions.objects;
+
+public enum PermissionMode {
```

Comment:
Should we make this a sub class of GuildPermission as theirs not a hole lot here?